contrib/init/sunos/gitea.xml

```diff
@@ -31,7 +31,7 @@
 		<exec_method type="method" name="stop"	exec=":kill" timeout_seconds="60"/>
 
 		<property_group name="application" type="application"></property_group>
-		<property_group name="startd" type="framework">
+		<property_group name="started" type="framework">
```

Comment:
Are you sure this change is right?

Reply:
please advise! I do not know `gitea` (yet).  But I know that `startd` is not a word ;-)

Reply:
I don't understand it either, but I think it can be googled:

https://www.google.com/search?q=sunos+framework+"startd"

![image](https://github.com/go-gitea/gitea/assets/2114189/6ed28c8f-0979-4899-b837-de531fec04fb)

Reply:
`d` stands for `daemon` in this case.

Reply:
Yup, it seems a predefined name in sunos, but I have no experience with sunos.

Reply:
ok, I will leave this `startd` alone